specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -153,10 +161,34 @@ Consider at least the below topics but you do not need to cover those that are n
       * Is it logged?
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
-### CICD
+### CI/CD
 
-#### e2e test image
-To run e2e tests alot of dependencies are needed (browsers, build systems etc...).
+CI/CD for frontend components leverages existing CICD for the VDK monorepo using
+GitLab.
+
+- [GitLab config for frontend components](/projects/frontend/cicd/.gitlab-ci.yml)
+- [@versatiledatakit/shared build script](/projects/frontend/cicd/build_shared.sh)
+- [@versatiledatakit/shared release script](/projects/frontend/cicd/publish_shared.sh)
+
+#### Dependency management
+
+`package-lock.json` is not included in version control for either package.
+Instead, for each pipeline run, `npm install` is run and `package-lock.json` is
+output as an artefact at the end. This ensures tracability for problems caused
+by dependency updates. It also ensures frontend components are compatible with
+the latest versions of their dependencies.
+
+#### Build and Test
+
+Opening a pull request with changes to `@versatiledatakit/shared` triggers a
```

Comment:
May correlate subproject directories (must be in project structure) rather than artifacts. For example, `cicd/` is not a part of any artifact, however is a valid changes directory.

Reply:
you resolved this, however I don't see any resolution -the statement seems confusing or simply wrong